frontend/src/metabase/nav/containers/MainNavbar/SidebarItems/DraggableSidebarLink.styled.tsx

```diff
@@ -15,8 +15,6 @@ export const DragIcon = styled(Icon)`
   cursor: grab;
 `;
 
-// Notice that dragged item styles are defined in sortable.css file
```

Comment:
I removed the comment entirely because those 2 files are removed. I'm not sure if we should leave any other comment here.